### ISSUE TITLE
Full op name to canonical op name

### ISSUE THREAD

**Issue by msaroufim** [meta-pytorch/BackendBench#121]
@manuelcandales recently audited https://github.com/meta-pytorch/BackendBench/pull/90 and let us know that are some pretty serious bugs there 

For example our intent is to say something like 

```
add_.Tensor -> add.Tensor
add.Tensor -> add.Tensor
```

But instead we end up being incorrect in cases such as 

1. for example max.dim_max is the out variant of max.dim, there is no out in the name
2. max.unary_out and max.dim

And so we should instead be looking whenever we can at the op._schemasa

```python
import torch
print(torch.ops.aten.max.dim_max._schema)
```

Instead his suggestion that we run something like the below, we don't need to depend on ET just capture what we need in a script

```python
>>> import torch
>>> from torchgen.model import SchemaKind
>>> from executorch.exir.dialects.edge.op.api import get_callable, to_variant
>>> op = get_callable("max.dim_max")
>>> canonical_op = to_variant(op, SchemaKind.functional)
>>> print(canonical_op)
aten.max.dim
```
